pytorch_lightning/core/lightning.py

```diff
@@ -1571,13 +1572,19 @@ def _auto_collect_arguments(cls, frame=None) -> Tuple[Dict, Dict]:
             parents_arguments.update(args)
         return self_arguments, parents_arguments
 
-    def save_hyperparameters(self, *args, frame=None) -> None:
-        """Save all model arguments.
+    def save_hyperparameters(
+        self, *args, ignore: Optional[Union[Sequence[str], str]] = None, frame: Optional[types.FrameType] = None
+    ) -> None:
+        """Save model arguments to ``hparams`` attribute.
 
         Args:
             args: single object of `dict`, `NameSpace` or `OmegaConf`
-             or string names or arguments from class `__init__`
+                or string names or arguments from class `__init__`
+            ignore: an argument or a list of arguments from class `__init__`
+                to be ignored
+            frame: a frame object. Default is None
 
+        Example::
         >>> class ManuallyArgsModel(LightningModule):
         ...     def __init__(self, arg1, arg2, arg3):
         ...         super().__init__()
```

Comment:
I think some indentation is required here to fix the docs.

Reply:
you can have the Example keyword up there. All you need is to indent the examples with a tab to avoid doc failures

Reply:
Updated!